ee/cis/win-10/cis-policy-queries.yml

```diff
@@ -2970,6 +2970,195 @@ spec:
 ---
 apiVersion: v1
 kind: policy
+spec:
+  name: >
+    CIS - Ensure 'Configure DNS over HTTPS (DoH) name resolution' is set to 'Enabled: Allow DoH' or higher
+  platforms: win10
+  platform: windows
+  description: |
+    This setting determines if DNS over HTTPS (DoH) is used by the system. DNS over HTTPS (DoH) is a protocol for performing remote Domain Name System (DNS) resolution over the Hypertext Transfer Protocol Secure (HTTPS). For additional information on DNS over HTTPS (DoH), visit: Secure DNS Client over HTTPS (DoH) on Windows Server 2022 | Microsoft Docs.
+    The recommended state for this setting is: 'Enabled: Allow DoH'. Configuring this setting to 'Enabled: Require DoH' also conforms to the benchmark.
+  resolution: |
+    To establish the recommended configuration via GP, set the following UI path to Enabled: Allow DoH (configuring to Enabled: Require DoH also conforms to the benchmark):
+    'Computer Configuration\Policies\Administrative Templates\Network\DNS Client\Configure DNS over HTTPS (DoH) name resolution'
```

Comment:
Is `Configure DNS over HTTPS (DoH) name resolution` another group policy that only shows up using Domain Controller? 

<img width="1179" alt="Screenshot 2023-03-16 at 6 43 50 AM" src="https://user-images.githubusercontent.com/71795832/225593315-6083019e-80c5-4412-8b3e-3c564b45dc2c.png">

Reply:
```
Note: This Group Policy path may not exist by default. It is provided by the Group Policy template DnsClient.admx/adml that is included with the Microsoft Windows 10 Release 21H2 Administrative Templates (or newer).
```

Reply:
For some reason it isn't included in the W10 policy definitions, even the most recent ones. I had to use the [W11 policy definitions](https://www.microsoft.com/en-us/download/104593), which loaded fine on W10:

![image](https://user-images.githubusercontent.com/954732/225706824-81974bd5-a86e-4e42-9e25-51292eff80e9.png)

Reply:
We're going to have a call this afternoon about how we want to file these edge cases, standby...

Reply:
Ok! Thanks @defensivedepth for all your work on these. I actually ran into 4 Window 11 policies in one of my PRs as well so the team met to be consistent on how we're filing these. Here are the notes:

![Screenshot 2023-03-16 at 4 45 50 PM](https://user-images.githubusercontent.com/71795832/225749163-ae470172-e544-4111-818e-65ca6a339af6.png)

To recap: 
We're going to file Win11 (missing on Win10) policies in cis-NOT-COMPLETED-policy-queries.yml instead of cis-policy-queries.yml
For the yaml fields, we will change the following: 
platform: win11
description: Include “This policy is meant for Windows 11.” on the first line of the description
tags: CIS_not_completed

Since you already have Win11 templates on your machine, I'll comb through these in the AM to help identify which ones are not on Win10.

Thanks for bearing with us 🧸

Reply:
Ok, just let me know if there is anything else I need to do for this PR. Thanks!

Reply:
> Ok, just let me know if there is anything else I need to do for this PR. Thanks!

Sweet, just left comments for you to revise, everything else looks great 🔥